src/reducers/splitThought.js

```diff
@@ -45,8 +48,10 @@ export default (state, { path, offset } = {}) => {
 
   // split the value into left and right parts
   const value = headValue(path)
-  const valueLeft = value.slice(0, offset)
-  const valueRight = value.slice(offset)
+
+  // Note: xhtmlPurifier adds unwanted <p> tags. So using strip to remove such tags
```

Comment:
Would be helpful to add a comment here *why* `xhtmlPurifier` is being used. I would recommend including the url to the issue as well. This will make it easy for future developers to trace the reasoning for this code, which would otherwise be quite mysterious!

Reply:
Sure will do that